VKApp/VKApp/Friends/FriendDetailCollectionViewController.swift

```diff
@@ -0,0 +1,37 @@
+// FriendDetailCollectionViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+///  Экран всех фотографий друга
+final class FriendDetailCollectionViewController: UICollectionViewController {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let friendDetailCellIdentifier = "friendDetailCell"
+    }
+
+    // MARK: - Public Property
+
+    var friend: Friends?
```

Comment:
```suggestion
    // MARK: - Public Property

    private var friend: Friends?
```
для разнообразия передадим данные через инит